hibernate-core/src/test/java/org/hibernate/orm/test/annotations/entity/Forest.java

```diff
@@ -58,6 +59,7 @@ public class Forest {
 	
 	@OptimisticLock(excluded=true)
 	@JdbcTypeCode( Types.LONGVARCHAR )
+	@Column(length = 10000)
```

Comment:
Actually, I believe that using the default length for `LONGVARCHAR` which should be `Length.LONG` is on purpose. Why do you have to change the size here? If `varchar` has a max capacity, it should be handled by implementing `Dialect#getMaxVarcharLength()`

Reply:
I understand that the maximum length for the `LONGVARCHAR ` type is set in the `getMaxVarcharLength()` method, which is correctly set to 32739 for Informix. The problem is that Informix also has a row length limit. The Forest entity is only used in the test `org.hibernate.orm.test.annotations.entity.BasicHibernateAnnotationsTest`, where I don't think the column length is critical.
```
create table Forest (
        id integer not null,
        length int8 not null,
        longDescription lvarchar(32600),
        bigText varchar(255),
        name varchar(255),
        smallText varchar(255),
        country blob,
        primary key (id)
    )" via JDBC [The operation causes a rowsize to exceed the allowable limit (32767).]
```

Of course, I could ignore these tests for InformixDialect, but I think more coverage is beneficial.

Reply:
You're right, thanks for double checking.